Public/Src/Cache/ContentStore/App/Application.cs

```diff
@@ -406,10 +400,29 @@ private void EnableRemoteTelemetryIfNeeded(string logFilePath)
                 logFilePath: logFilePath + TmpCsvLogFileExt,
                 serviceName: ServiceName,
                 schema: KustoTableCsvSchema,
+                renderConstColums: false,
                 severity: _fileLogSeverity,
                 maxFileSize: _csvLogMaxFileSize
                 );
 
+            var indexedColumns = _csvFileLog.FileSchema.Select((col, idx) => new CsvColumnMapping { ColumnName = col.ToString(), Ordinal = idx });
+            var constColumns = _csvFileLog.ConstSchema.Select(col => new CsvColumnMapping { ColumnName = col.ToString(), ConstValue = _csvFileLog.RenderConstColumn(col) });
+            var csvMapping = indexedColumns.Concat(constColumns).ToArray();
+
+            var csvMappingStr = string.Join("", csvMapping.Select(col => $"{Environment.NewLine}  Name: '{col.ColumnName}', ConstValue: '{col.ConstValue}', Ordinal: {col.Ordinal}"));
+            _logger.Always("Using csv mapping:{0}", csvMappingStr);

```

Comment:
Nit: space?

Reply:
`csvMappingStr` starts with a new line, so the omission of space there was intentional.

Reply:
Just saw it, sorry 👍